lib/health_checking/health_checks/board/issue_blocked_label_check.rb

```diff
@@ -0,0 +1,31 @@
+module HealthChecking
+  module HealthChecks
+    module Board
+      class IssueBlockedLabelCheck
+        include HealthCheck
+        name 'issue_blocked_label_check'
+        weight :warning
+        authorization :collaborator
+        passed "Issue blocked label is present"
+        failed "Issue blocked label is not present"
```

Comment:
Yeah, that copy is better, issue is redundant. 

> Or we could add some sort of tooltip to give a help bubble with more explanation.

I think for our UX 'funnel' phase we should identify things like this and determine a good way to provide intuitive info about HuBoard on the fly